lib/rpmsg/rpmsg_virtio.c

```diff
@@ -545,6 +545,14 @@ int rpmsg_init_vdev(struct rpmsg_virtio_device *rvdev,
 	rdev->ns_bind_cb = ns_bind_cb;
 	vdev->priv = rvdev;
 	rdev->ops.send_offchannel_raw = rpmsg_virtio_send_offchannel_raw;
+
+	if (vdev->features & (1 << VIRTIO_RPMSG_F_BUFSZ)) {
+		rpmsg_virtio_read_config(rvdev, 0,
+					 &rvdev->shbuf_size,
```

Comment:
It looks like you assume offset 0 is the shared buffer size in virito config space.
Better to make it macro in rpmsg_virito.h file.

Reply:
Sure, I will add it